js/src/lib/Details/Cdn.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { Di } from './';
+
+const JsDelivr = ({ name }) => (
+  <Di
+    title="jsDelivr"
+    description={
+      <a
+        href={`https://cdn.jsdelivr.net/npm/${name}/`}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        cdn.jsdelivr.net/npm/{name}/
+      </a>
+    }
+  />
+);
+
+class Cdn extends Component {
+  render() {
+    const {
+      name,
+    } = this.props;
+
+    return (
+        <article className="details-side--cdns">
+            <h1>{window.i18n.detail.cdns}</h1>
+            <dl>
+                <JsDelivr name={name} />
```

Comment:
Just do `name={this.props.name}`, no advantage in spreading the object if it's only used in a single place.